ethpm_cli/scraper.py

```diff
@@ -198,14 +242,23 @@ def process_entries(
 ) -> Iterable[Tuple[str, str]]:
     for entry in all_entries:
         if entry["address"] == address:
+            logger.info(
+                "<Package %s==%s> released on registry @ %s.\n" "Manifest URI: %s\n",
+                entry["args"]["packageName"],
+                entry["args"]["version"],
+                address,
```

Comment:
Doesn't have to be for this PR but doing a reverse resolution on this address to see if it is an ENS domain would be really good UX (still should probably display the address, but you could *humanize* it and truncate the middle in that case)